src/corner_kick/scripts/start_test.sh

```diff
@@ -7,4 +7,5 @@ DIRECTORY=`dirname $0`
 # Move to this directory and start the test suite of
 # Corner Kick.
 cd $DIRECTORY
-yarn test
+chmod+x
```

Comment:
No. That's not how you fix permissions. You need to run `chmod +x start_test.sh` ON the file, not write it IN the file. From the scripts directory, open a console and run `chmod +x start_test.sh` then commit.

Reply:
Fixed this one. 
Is the command for pulling from upstream "git fetch upstream"?